tools/project_manager/editproject.php

```diff
@@ -275,12 +275,12 @@ public function set_from_post()
             }
         }
 
-        // postednum needs to either be a valid number or null, not an empty string
+        // postednum needs to either be a valid int or null, not an empty string
         if (isset($_POST["postednum"])) {
             if ($_POST["postednum"] == "") {
                 $this->project->postednum = null;
             } else {
-                $this->project->postednum = sprintf("%d", $_POST["postednum"]);
+                $this->project->postednum = (int)$_POST["postednum"];
```

Comment:
Is it worth using `get_integer_param` here? I think this would be equivalent:

```suggestion
                $this->project->postednum = get_integer_param($_POST, "postednum");
```

Reply:
It isn't quite that straightforward as `get_integer_param()` requires a default, a min, and a max. I think in this particular case its better to take the number provided and just make it an int as we don't control the range of numbers PG provides.